### PR TITLE
Changed text to align with Readme formatting

### PR DESCRIPTION
**Pull request by ybasford**
# What does this PR do?

Fixes #`put issue number here`.

Add any further explanations here.

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [ ] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: docs/content/setup/_index.md [hunk 1]
```diff
@@ -9,18 +9,19 @@ tags = ["setup"]
...
+- [Docker](https://techdocs.akamai.com/infection-monkey/docs/docker)
+- [Linux](https://techdocs.akamai.com/infection-monkey/docs/linux)
+- [AWS](https://techdocs.akamai.com/infection-monkey/docs/aws)
+- [Azure](https://techdocs.akamai.com/infection-monkey/docs/azure)
+- [Accounts and security](https://techdocs.akamai.com/infection-monkey/docs/accounts-and-security)
 
-{{% notice tip %}}
-You can find information about [operating system compatibility and support here](../reference/operating_systems_support).
-{{% /notice %}}
+After setting the Monkey up, check out our [Getting started guide](doc:getting-started).
```
Comment (mssalvatore) [inline docs/content/setup/_index.md:22]: Similarly, this link doesn't work in our development platform, which is problematic for us. Is there a specific tool we should be using to render the markdown during development?

File: docs/content/setup/aws.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ weight: 5
 tags: ["setup", "aws"]
 ---
 
-## Deployment
+# Deployment
```
Comment (mssalvatore) [inline docs/content/setup/aws.md:10]: This change makes `Deployment` appear to be the same size as the title.

**Review by mssalvatore — CHANGES_REQUESTED** [review]

**Comment by ilija-lazoroski** [comment]
@ybasford can we rebase this on develop ?

**Comment by mssalvatore** [comment]
Happy New Year! This PR hasn't seen any activity since August. I'm closing this due to inactivity. We can reopen if this effort gets revived.

**Pull request closed without merging**